about/citations/index.markdown

```diff
@@ -14,13 +14,13 @@ title: About
           <p>David Coleman, Ioan A. Șucan, Sachin Chitta, Nikolaus Correll,
           Reducing the Barrier to Entry of Complex Robotic Software: a MoveIt! Case Study,
           <em>Journal of Software Engineering for Robotics</em>,
-          5(1):3–16, May 2014. doi: <a href="http://dx.doi.org/10.6092/JOSER_2014_05_01_p3">10.6092/JOSER_2014_05_01_p3</a>.</p>
-          <p class="text-center"><a class="button" href="https://arxiv.org/pdf/1404.3785">PDF</a>
-          <a class="button" href="http://dx.doi.org/10.6092/JOSER_2014_05_01_p3">Publisher</a></p>
+          5(1):3–16, May 2014. doi: <a href="http://dx.doi.org/10.6092/JOSER_2014_05_01_p3" target="_blank">10.6092/JOSER_2014_05_01_p3</a>.</p>
+          <p class="text-center"><a class="button" href="https://arxiv.org/pdf/1404.3785" target="_blank">PDF</a>
+          <a class="button" href="http://dx.doi.org/10.6092/JOSER_2014_05_01_p3" target="_blank">Publisher</a></p>
         </div>
         <div class="main-card-single boarder main-card-single-padding">
           <img class="mx-auto d-block quote-img" src="/assets/images/people_page/quote.png">
-          <p>Ioan A. Sucan and Sachin Chitta, "MoveIt", [Online] Available at <a href="https://moveit.ros.org">moveit.ros.org</a>.</p>
+          <p>Ioan A. Sucan and Sachin Chitta, "MoveIt", [Online] Available at <a href="/">moveit.ros.org</a>.</p>
```

Comment:
@auman66 if someone copy pastes this to their document, with the hyperlink be broken?

Reply:
Good point. Fixed